aten/src/ATen/native/native_functions.yaml

```diff
@@ -11238,13 +11238,13 @@
   python_module: linalg
   variants: function
 
-- func: linalg_qr(Tensor self, str mode='reduced') -> (Tensor Q, Tensor R)
+- func: linalg_qr(Tensor A, str mode='reduced') -> (Tensor Q, Tensor R)
```

Comment:
So, we prefer a reduced form of QR while SVD for some reason is full :)

Reply:
I KNOW RIGHT??!!!! -.-